src/algebra/group_ring_action.lean

```diff
@@ -65,6 +65,14 @@ def mul_semiring_action.to_ring_equiv [mul_semiring_action G R] (x : G) : R ≃+
 { .. distrib_mul_action.to_add_equiv R x,
   .. mul_semiring_action.to_ring_hom G R x }
 
+/-- Compose a `mul_semiring_action` with a `monoid_hom`, with action `f r' • m`.
+See note [reducible non-instances]. -/
+@[reducible] def mul_semiring_action.comp_hom (g : K →* M) [mul_semiring_action M R] :
+  mul_semiring_action K R :=
+{ smul := has_smul.comp.smul g,
+  ..distrib_mul_action.comp_hom R g,
+  ..mul_distrib_mul_action.comp_hom R g }
```

Comment:
`K` and `M` should be implicit arguments here; I think right now they're explicit, which is unecessary since both can be inferred from `g`.

Reply:
I am not sure what you mean in this case - I get errors by making them implicit.
Please let me know how I can make this change

Reply:
You need to make them implicit for just this definition. Can you push the code that gives errors so that I can help you?